packages/colorpicker/pubspec.yaml

```diff
@@ -0,0 +1,49 @@
+name: colorpicker
+description: "A new Flutter package project."
+version: 0.0.1
+homepage:
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
+  flutter: ">=1.17.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^2.0.0
+  build_runner: ^2.2.0
+
+flutter:
+  uses-material-design: true
+  assets:
+    - assets/img/
```

Comment:
When clicking the color in bottom nav bar I get:

Another exception was thrown: Unable to load asset: "assets/img/checkerboard.png".

Maybe an issue with this? Could be somewhere else too.

Also if I were coding this I would import components library to get access to CheckerboardImg from there. Now it is a duplicate image.